<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>英语听写小程序</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget {
	
	background-color: rgb(238, 239, 255);
}

#MainWindow *{
	font: 10pt &quot;楷体&quot;;
}
#text, #text2 {
	
	background-color: rgb(255, 255, 247);
	font: 12pt &quot;楷体&quot;;
}
#tab,#tab_2,#tab_3 {
	
	background-color: rgb(221, 255, 234);
}
#inputbutton:hover,#showbutton:hover,#startbutton:hover,#showhistorybutton:hover,#filebutton:hover,#savebutton:hover,#readaloudbutton:hover,#writebutton:hover {
	color: black;
	background-color: rgb(219, 219, 255);
}

#inputbutton:hover,#showbutton:hover,#startbutton:hover,#showhistorybutton:hover,#filebutton:hover,#savebutton:hover,#readaloudbutton:hover,#writebutton:hover {
	background-color: rgb(168, 160, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_17">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>听写界面</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1,2">
                <item>
                 <widget class="QLabel" name="label1">
                  <property name="text">
                   <string>听写的个数</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="numbox"/>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,1,2">
                <item>
                 <widget class="QLabel" name="label3">
                  <property name="text">
                   <string>发音次数</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="repeatnumbox"/>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,1,2">
                <item>
                 <widget class="QLabel" name="label2">
                  <property name="text">
                   <string>单词间隔</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="timebox">
                  <property name="styleSheet">
                   <string notr="true">#inputbutton {
	background-color: rgb(208, 211, 255)
}</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>英语发音类型</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="enbutton">
                  <property name="text">
                   <string>英式发音</string>
                  </property>
                  <attribute name="buttonGroup">
                   <string notr="true">buttonGroup_2</string>
                  </attribute>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="usabutton">
                  <property name="text">
                   <string>美式发音</string>
                  </property>
                  <attribute name="buttonGroup">
                   <string notr="true">buttonGroup_2</string>
                  </attribute>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>中文发音类型</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QGridLayout" name="gridLayout">
                  <item row="0" column="0">
                   <widget class="QRadioButton" name="radioButton">
                    <property name="text">
                     <string>女声</string>
                    </property>
                    <attribute name="buttonGroup">
                     <string notr="true">buttonGroup</string>
                    </attribute>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QRadioButton" name="radioButton_2">
                    <property name="text">
                     <string>男声</string>
                    </property>
                    <attribute name="buttonGroup">
                     <string notr="true">buttonGroup</string>
                    </attribute>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QRadioButton" name="radioButton_3">
                    <property name="text">
                     <string>度逍遥</string>
                    </property>
                    <attribute name="buttonGroup">
                     <string notr="true">buttonGroup</string>
                    </attribute>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QRadioButton" name="radioButton_4">
                    <property name="text">
                     <string>度丫丫</string>
                    </property>
                    <attribute name="buttonGroup">
                     <string notr="true">buttonGroup</string>
                    </attribute>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QPushButton" name="inputbutton">
                <property name="text">
                 <string>导入单词</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="writebutton">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>手动导入</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="readaloudbutton">
                <property name="text">
                 <string>朗读单词</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="showbutton">
                <property name="text">
                 <string>查看单词</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="startbutton">
                <property name="text">
                 <string>开始听写</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>输出框</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_16">
             <item>
              <widget class="QPlainTextEdit" name="text">
               <property name="maximumBlockCount">
                <number>0</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>历史记录</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>路径 </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="fileedit2"/>
            </item>
            <item>
             <widget class="QPushButton" name="showhistorybutton">
              <property name="text">
               <string>浏览</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_19" stretch="2,1,1,2">
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>单词个数</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="numbutton">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTextBrowser" name="text2"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>设置</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,1,0,1,0,1,0,1,0,1,0,1,0,1,0,1">
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>单词表选择</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="fileedit">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="filebutton">
                <property name="text">
                 <string>预览</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>听写个数</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="defaultnumBox">
                <property name="value">
                 <number>0</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>发音个数</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="defaultrepeatbox"/>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>发音间隔</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="defaultvoiceintervalbox"/>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_5">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>单词间隔</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="defaultwordintervalbox"/>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_6">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>中文语音速度</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="speadbox">
                <property name="minimumSize">
                 <size>
                  <width>49</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximum">
                 <number>9</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_7">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>英文发音类型</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QRadioButton" name="defaultenbutton">
                  <property name="text">
                   <string>英式发音</string>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                  <property name="autoRepeat">
                   <bool>false</bool>
                  </property>
                  <property name="autoExclusive">
                   <bool>true</bool>
                  </property>
                  <attribute name="buttonGroup">
                   <string notr="true">buttonGroup_3</string>
                  </attribute>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="defaultusabutton">
                  <property name="text">
                   <string>美式发音</string>
                  </property>
                  <attribute name="buttonGroup">
                   <string notr="true">buttonGroup_3</string>
                  </attribute>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_8">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_13">
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>中文发音类型</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="1" column="0">
                 <widget class="QRadioButton" name="defaultdubutton">
                  <property name="text">
                   <string>度逍遥</string>
                  </property>
                  <attribute name="buttonGroup">
                   <string notr="true">buttonGroup_4</string>
                  </attribute>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QRadioButton" name="defaultmalebutton">
                  <property name="text">
                   <string>男声</string>
                  </property>
                  <attribute name="buttonGroup">
                   <string notr="true">buttonGroup_4</string>
                  </attribute>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QRadioButton" name="defaultfemalebutton">
                  <property name="text">
                   <string>女声</string>
                  </property>
                  <attribute name="buttonGroup">
                   <string notr="true">buttonGroup_4</string>
                  </attribute>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QRadioButton" name="defaultduyayabutton">
                  <property name="text">
                   <string>度丫丫</string>
                  </property>
                  <attribute name="buttonGroup">
                   <string notr="true">buttonGroup_4</string>
                  </attribute>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_9">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="2,1,2">
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="savebutton">
              <property name="text">
               <string>保存</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_4"/>
  <buttongroup name="buttonGroup_3"/>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
